tests/inst/src/sysroot.rs

```diff
@@ -45,3 +48,13 @@ fn test_immutable_bit() -> Result<()> {
     cmd_has_output(sh_inline::bash_command!("lsattr -d /").unwrap(), "-i-")?;
     Ok(())
 }
+
+#[itest]
+fn test_tmpfiles() -> Result<()> {
+    if skip_non_ostree_host() {
+        return Ok(());
+    }
+    let metadata = Path::new("/run/ostree").metadata()?;
+    assert_eq!(metadata.permissions().mode(), 0o755);
```

Comment:
This failed in CI with:
```
kola-runext-ostree-test[1578]: thread '<unnamed>' panicked at 'assertion failed: `(left == right)`
kola-runext-ostree-test[1578]:   left: `16877`,
kola-runext-ostree-test[1578]:  right: `493`', tests/inst/src/sysroot.rs:58:5
```

Reply:
Always nice to get those CI failures that let you know CI is working.